skycultures/indian/constellation_names.eng.fab

```diff
@@ -19,49 +20,85 @@
 # If your skyculture is not a variant of the generally known Western skyculture, please include an
 # English translation to the name given in the native language. Else translators will not be able to
 # translate the name
+#################
+
+# For the column 1: An abbreviation for the constellation. (The "abbreviated" mode should allow to use short unique markers, without filling the screen with text. In scientific context, the 88 IAU constellations are abbreviated with 3 characters, which is just enough to remember the constellation.)
+# For the column 2: We use IAST transliteration for unambious transliteration of devanAgarI using latin alphabets, while using phonetically more intuitive equivalents in suggested translations.
+# For column 3: We've referred to highly regarded sanskrit to sanskrit dictionaries kalpadruma and vAchaspatyam for etymologies below.
 
-# We use IAST transliteration for unambious transliteration of devanAgarI using latin alphabets, while using phonetically more intuitive equivalents in suggested translations.
+#################
+# Nakshatra-s.
+#################
+# We show Nakṣatra-s associated with a single star as a star, rather than as a constellation.
+# From @gzotti: "if some Nakshatras may receive sky art, all Nakshatras should be coded as constellations. When we do the change mentioned above, those will then be separated (or just marked inside the constellation file). ... We may highlight the 1-star constellation in later editions e.g. by a circle." 
 
-## We've referred to highly regarded sanskrit to sanskrit dictionaries kalpadruma and vAchaspatyam for etymologies below.
+Aśv	"Aśvinī"	_("Divine physician twins")
```

Comment:
minor comment, personal preference: I find it odd to see diacritical marks mixed with first letter capital (though i can still read it unambiguously). my preference is for small-caps with diacritics.

Reply:
I myself am ambivalent - but standard practice seems to be to capitalize initial letter of proper nouns.
Thanks for drawing my attention there though - Ensured that initial letter is always capital for consistency.